getting_started.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Getting Started"
+nav_order: 2
+description: >
+    Getting Started with FalkorDB Graph Database.
+---
+
+
+# Getting Started with FalkorDB
+
+This guide will walk you through setting up FalkorDB, modeling a social network as a graph, 
+and accessing it using the [FalkorDB Python client](/clients) with [Cypher](/cypher).
+
+---
+
+## Prerequisites
+
+1. **FalkorDB Instance**: Set up FalkorDB (on-prem or cloud). [Learn More](/configuration).  
+2. **Python Installed**: Ensure you have Python 3.8+ installed.  
+3. **Install FalkorDB Python Client**:
+   
+   ```bash
+   pip install falkordb
+   ```
+   
+---
+
+## Step 1: Model a Social Network as a Graph
+
+Let's create a simple graph for a social network where:  
+- **Nodes** represent `User` and `Post`.
+- **Relationships** represent `FRIENDS_WITH` and `CREATED`.
+
+### Graph Schema
+
+| Node Type | Properties               |
+|-----------|--------------------------|
+| User      | `id`, `name`, `email`    |
+| Post      | `id`, `content`, `date`  |
+
+| Relationship Type | Start Node | End Node | Properties   |
+|-------------------|------------|----------|--------------|
+| FRIENDS_WITH      | User       | User     | `since`      |
+| CREATED           | User       | Post     | `time`       |
+
+---
+
+## Step 2: Load Data into FalkorDB
+
+Here’s how you can model and load the data.
+
+### Cypher Query to Create the Data
+
+```cypher
+CREATE (alice:User {id: 1, name: "Alice", email: "alice@example.com"})
+CREATE (bob:User {id: 2, name: "Bob", email: "bob@example.com"})
+CREATE (charlie:User {id: 3, name: "Charlie", email: "charlie@example.com"})
+
+CREATE (post1:Post {id: 101, content: "Hello World!", date: 1701388800})
+CREATE (post2:Post {id: 102, content: "Graph Databases are awesome!", date: 1701475200})
+
+CREATE (alice)-[:FRIENDS_WITH {since: 1640995200}]->(bob)
+CREATE (bob)-[:FRIENDS_WITH {since: 1684108800}]->(charlie)
+CREATE (alice)-[:CREATED {time: 1701388800}]->(post1)
+CREATE (bob)-[:CREATED {time: 1701475200}]->(post2)
```

Comment:
date and time as they are right now can be confusing, either use Python to create timestamps from actual dates or consider changing the attribute to something else which doesn't require time / date datatype

Reply:
how would you do it?

Reply:
Look up Python date / time to unix timestamp

Reply:
I looked it up, it will bloat the example too much